NGitLab/Models/Membership.cs

```diff
@@ -41,5 +44,11 @@ public class Membership
         /// </summary>
         [DataMember(Name = "access_level")]
         public int AccessLevel;
+
+        /// <summary>
+        /// Membership expiration date
+        /// </summary>
+        [DataMember(Name = "expires_at")]
+        public DateTime? ExpiresAt;
```

Comment:
I have a few questions, knowing that this class inappropriately
- provides public fields rather than properties
- uses `DateTime`

At the risk of introducing breaking changes, should we
- convert fields to properties
- use `DateTimeOffset` throughout this class instead of `DateTime`?

@meziantou , what's your take on this? Can we afford that API change?

Reply:
We should avoid breaking changes if possible!

> provides public fields rather than properties

New classes use properties, older ones use fields. We try to not mix both in the same class but maybe there are some inconsistencies in the project.

Reply:
Is there a risk of breaking someone's code by switching the fields to properties if those properties all have getters and setters?

Reply:
Ok, so for now we'll keep using fields here... 😉  

However, the new field can probably be a `DateTimeOffset`. We use that type elsewhere and I'm fairly certain there is no JSON deserialization issues.

Reply:
> Is there a risk of breaking someone's code by switching the fields to properties if those properties all have getters and setters?

Unfortunately, yes. For now we'll keep using fields here... 😉  

However, the new field can probably be a `DateTimeOffset`. We use that type elsewhere and I'm fairly certain there is no JSON deserialization issues.

Reply:
Is it not better to keep it consistent with the `CreatedAt` Attribute?

Reply:
Yeah, ok. We'll revisit all `DateTime`s at a later time, when a planned refactoring is done.